app/components/m-header.hbs

```diff
@@ -1,87 +1,160 @@
-              {{#link-to "settings.overview"}}
-                <span class="auk-tabs__tab-inner">{{t "settings"}}</span>
-              {{/link-to}}
+              <LinkTo @route="settings.overview">
+                <span class="auk-tabs__tab-inner">
+                  <span class="auk-tabs__label">
+                    {{t "settings"}}
+                  </span>
+                </span>
+              </LinkTo>
             </li>
           {{/if}}
         </ul>
       </div>
     </div>
     <div class="auk-toolbar-complex__right">
-      <div class="auk-toolbar-complex__right">
-        {{#if showEnvironmentName}}
-          <div class="vlc-environment-pill {{environmentClass}}">
-            {{environmentName}}
-            {{capitalize (t "environment")}}
+      {{#if this.showEnvironmentName}}
+        <div class="auk-toolbar-complex__item auk-u-mr-3">
+          <div class="vlc-environment-pill {{this.environmentClass}}">
+            {{this.environmentName}}
           </div>
-        {{/if}}
-      </div>
-      <div class="auk-toolbar-complex__item">
-        <div class="auk-u-flex auk-u-flex--vertical-center auk-u-text-small auk-u-mx-2">
-          {{#if currentSession.userContent}}
-            <div class="auk-u-mr">
-              {{currentSession.userContent.firstName}}
-              {{currentSession.userContent.lastName}}
-              ({{currentSession.userRole}})
-            </div>
-            <a href="" {{action "logout"}}>
-              {{t "sign-out"}}
-            </a>
-          {{else}}
-            {{#link-to "login"}}
-              {{t "login"}}
-            {{/link-to}}
-          {{/if}}
         </div>
+      {{/if}}
+      <div class="auk-toolbar-complex__item">
+        <WebComponents::AuButtonLink
+                @skin="muted"
+                @icon="ki ki-circle-question">
+          {{t "help"}}
+        </WebComponents::AuButtonLink>
+        <AttachPopover
+                @renderInPlace={{true}}
+                @class="ember-attacher-popper"
+                @hideOn="clickout click"
+                @showOn="click"
+                @animation="shift"
+                @placement="bottom" {{!-- as |attacher| --}}
+        >
+          <WebComponents::AuDropdown>
+            <WebComponents::AuDropdown::Item @href="https://kaleidos.vlaanderen.be/Kaleidos_Raadplegen_20200629.pdf"
+                                             target="_blank">
+              {{t "look-at-documentation"}}
+            </WebComponents::AuDropdown::Item>
+            <WebComponents::AuDropdown::Item @href="mailto:kaleidos.servicedesk@vlaanderen.be">
+              <div>
+                <span>{{t "contact-us"}}</span>
+                <span class="auk-u-muted auk-u-text-small">{{t "email-servicedesk"}}</span>
+              </div>
+            </WebComponents::AuDropdown::Item>
+          </WebComponents::AuDropdown>
+        </AttachPopover>
+      </div>
+      <div class="auk-toolbar-complex__item auk-u-mr-3">
```

Comment:
<WebComponents::AuToolbar::Item>

Reply:
die auk-u-mr-3 is hier wel nodig, doe dan
```<WebComponents::AuToolbar::Item class="auk-u-mr-3">```
auToolbar/item heeft ...attributes, dus class meegeven zoals dit werkt dan in uitzonderingen